docs/security/authorization.md

```diff
@@ -8,38 +8,36 @@
 
 <h2 id="1">Authorization Modes</h2>
 
-Three primary modes for Kyuubi authorization are available by [spark-authorizer](https://github.com/yaooqinn/spark-authorizer):
+Three primary modes for Kyuubi authorization are available by [Submarine Spark Security](https://github.com/apache/submarine/tree/master/submarine-security/spark-security):
```

Comment:
use `https://github.com/apache/submarine/tree/master/submarine-security/spark-security` to replace `https://github.com/yaooqinn/spark-authorizer`.